README.md

```diff
@@ -1 +1,55 @@
-# perl-Mojo-WebSocketProxy
\ No newline at end of file
+# perl-Mojo-WebSocketProxy
+[![Build Status](https://travis-ci.org/binary-com/perl-Mojo-WebSocketProxy.svg?branch=master)](https://travis-ci.org/binary-com/perl-Mojo-WebSocketProxy) 
+
+Using this module you can forward websocket JSON-RPC 2.0 requests to RPC server.
```

Comment:
More detailed docs will be available on CPAN, from [this](https://github.com/binary-com/perl-Mojo-WebSocketProxy/pull/1/files#diff-cef47db8734c648526c6fe7587f28f9eR15) POD.
I wanted to paste a link to CPAN in README. 
Should I duplicate that doc in README?